public/templates/teacher_sets/detail.html

```diff
@@ -13,13 +13,13 @@ <h3>{{ts.title}}</h3>
         <div id='total_available_copies' align="center" style="font-weight:bold"> {{ts.available_copies}} of 
           {{ts.total_copies}} Available | {{ts.new_holds_count}} sets pending
         </div><p style="white-space: pre"></p>
-        <div data-ng-show="ts.available_copies > 0 && !active_hold && is_available ">
+        <div data-ng-show="ts.is_copies_available && is_available ">
```

Comment:
how is is_available different from is_copies_available?

Reply:
> how is is_available different from is_copies_available?
@daryanypl (is_copies_available  value calculating from available_copies count and is_available calculating  from  availability string)

Reply:
yes, but that string is itself calculated in the item update code that you wrote before.  and it's dependent on the # of copies available.  basically, I'm not sure you need all three variables.